### ISSUE TITLE
The onAnimationComplete prop does not get fired

### ISSUE THREAD

**Issue by deeptianand** [RobertFOConnor/react-native-progress-wheel#16]
**Describe the bug**
Hi! I was under the impression that the onAnimationComplete callback would get fired once the animation was complete but I do not see this happening. Am I understanding this correctly?

**To Reproduce**
```
<AnimatedProgressWheel
          progress={100}
          animateFromValue={0}
          duration={60000}
          size={50}
          width={6}
          onAnimationComplete={() => {console.log("Animation is complete!")}}
        ></AnimatedProgressWheel>
```
**Expected behavior**
Animation is complete appears in the console

**Comment by deeptianand** [comment]
This is probably because there hasn't been a new release for this package. Wondering if you were planning to do so soon?

**Comment by RobertFOConnor** [comment]
Working on it, expect it in soon.

**Comment by RobertOConnorMM** [comment]
Fixed in version 1.0.5.

**Comment by RobertFOConnor** [comment]
Fixed in version 1.0.5.
